### PR TITLE
Update to webpack 5 to resolve dep security issues

### PR DESCRIPTION
**Pull request by johnBartos**
*Issue #, if available:*

*Description of changes:*
- Updates packages to resolve security warnings from NPM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "lint": "tsc --noEmit --project ./tsconfig.json",
     "bundle": "npm-run-all lint build",
     "watch:tsc": "tsc-watch --project ./tsconfig.json --noEmit",
-    "watch:webpack": "webpack-dev-server --config ./webpack.config.js --watch --compress",
+    "watch:webpack": "webpack-dev-server --config ./webpack.config.js --compress",
```
Comment (slawrence) [inline package.json:12]: Was this intended?

**Review by slawrence — APPROVED** [review]
LGTM. A spot check of formatting changes did not appear to modify logic.

File: package.json (hunk 1 above)
Comment (johnBartos) [reply package.json:12]: Yup, it's no longer a valid option for webpack 5

Comment (johnBartos) [reply package.json:12]: Watching happens by default as far as I can tell

**Pull request merged**